### ISSUE TITLE
Reference issue

### ISSUE THREAD

**Issue by waqas1223** [Vitaliy-1/DOCX2JATS#8]
Hi vitaliy that plugin DOCX2JATS is working prefect but i am facing a issue is in refernce.
in cmd command show reffernce list is added but cant display on journal.
i doall your instructin
refernce tag haeading 
AMA stlye
Cite from mendeley
[Article1.docx](https://github.com/Vitaliy-1/DOCX2JATS/files/6646506/Article1.docx)

**Comment by Vitaliy-1** [comment]
Hi @waqas1223,

Can you describe what tools (and their versions) did you use to generate this DOCX file? I don't see any citation tags generated by Mendeley in this document.

**Comment by waqas1223** [comment]
I use docx2jats through cmd follow this video

https://acahya.web.id/tutorial-generate-xml-galley-ojs-3-1/
 and ojs version 3.2
in my pc this document showall citation of this document through Mendeley

**Comment by Vitaliy-1** [comment]
@waqas1223,

There are 2 ways right now for references to be parsed correctly, either using Mendeley plugin for MS Word/LibreOffice Writer or Zotero plugin for MS Word. Both of those plugins require correspondent desktop apps to be installed on the system. They allow exporting DOCX file with specific markup for references, which then can be parsed by DOCX Converter Plugin.

**Comment by waqas1223** [comment]
yes sir i read all you instruction before conversion . so in this article i cite all reference through web Mendeley then add in word through  Mendeley plugin.
can you send a complete word file which i convert  though cmd .

**Comment by Vitaliy-1** [comment]
@waqas1223,

Sorry, haven't noticed where the issue is opened. This is the old version of a converter written in Java. Converter much evolved from that time. The command-line version of a new one is here: https://github.com/Vitaliy-1/docxToJats. And OJS plugin is here: https://github.com/Vitaliy-1/docxConverter.

**Comment by waqas1223** [comment]
ok thanks for reply i will go through that and will let you  know soon and this time i try to cite with zetero.
